examples/grover/grover.cpp

```diff
@@ -0,0 +1,75 @@
+#include <tweedledum/tweedledum.hpp>
+#include <iostream>
+#include <cstdint>
+#include <string>
+#include <unordered_map>
+#include <utility>
+
+#include <fmt/format.h>
+/*
+    This file implement grover search example for 4 qubits to find '1101'
+*/
+
+int main(int argc , char** argv)
+{
+
+    (void) argc;
+	(void) argv;
```

Comment:
Indentation guidelines in the showcase are 2 spaces.

Reply:
That indentation of scopes should be done using 2 spaces. Right now, it's a mix between tabs and 4 spaces.